include/flux/core/inline_sequence_base.hpp

```diff
@@ -271,6 +272,15 @@ struct inline_sequence_base {
     [[nodiscard]]
     constexpr auto filter(Pred pred) &&;
 
+    template <typename Func>
+    requires (std::invocable<Func&, element_t<Derived>> &&
+        detail::optional_like<std::invoke_result_t<Func&, element_t<Derived>>>)
```

Comment:
Formatting: indent the requires clause

```suggestion
        requires (std::invocable<Func&, element_t<Derived>> &&
            detail::optional_like<std::invoke_result_t<Func&, element_t<Derived>>>)
```